presto-main/src/main/java/com/facebook/presto/operator/aggregation/state/NullableSliceState.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.state;
+
+import io.airlift.slice.Slice;
+
+@AccumulatorStateMetadata(stateSerializerClass = NullableSliceStateSerializer.class)
+public interface NullableSliceState
+        extends AccumulatorState
+{
+    @InitialBooleanValue(true)
+    boolean isNull();
```

Comment:
I don't think you need a boolean here, since you can just check if `getSlice()` returns `null`